scripts/rv_expression_association/get_rv_outliers.py

```diff
@@ -0,0 +1,166 @@
+import click
+import logging
+import hail as hl
+import pandas as pd
+from cpg_utils.hail_batch import output_path
+from cloudpathlib import AnyPath
+
+# use logging to print statements, display at info level
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+
+# get OneK1K sample ID and gene name as arguments using click
+@click.command()
+@click.option('--onek1k-id', required=True)
+@click.option('--gene-name', required=True)
+@click.option('--chrom', required=True)
+def main(
+    onek1k_id: str,
+    gene_name: str,
+    chrom: str,
+):
+    """for a given individual,
+    and gene for which that individual is an expression outlier
+    get all variants within a window,
+
+    then filter for variants that:
+    - are alt (0/1 or 1/1) for that individual
+    - are biallelic SNVs
+    - have regulatory consequences based on VEP annotations
+
+    also annotate variants with the following:
+    - CADD score
+    - MAF within the OneK1K sample
+    - MAF in gnomad
+
+    create the following table:
+    donor ID | gene ID | variant ID | position | CADD | MAF (OneK1K) | MAF (gnomad) | ..
+    """
+
+    # file matching OneK1K IDs to CPG (internal) and TOB (external) IDs
+    sample_key_filename = (
+        'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/OneK1K_CPG_IDs.tsv'
+    )
+    sample_key_df = pd.read_csv(sample_key_filename, sep='\t', index_col=0)
+
+    cpg_id = sample_key_df[sample_key_df['OneK1K_ID'] == onek1k_id]['InternalID']
+    logging.info('CPG ID: {}'.format(cpg_id))  # e.g., 'CPG9951'
+
+    # define output filename and check if it already exists
+    output_filename = AnyPath(output_path(f'{cpg_id}_{gene_name}.csv'))
+    logging.info('Output file: {}'.format(output_filename))
+
+    # skip if file already exists
+    if output_filename.exists():
+        raise Exception(f'File {output_filename} already exists, exiting')
+
+    # get VEP-annotated WGS object (hail matrix table)
+    mt = hl.read_matrix_table('gs://cpg-tob-wgs-test/tob_wgs_vep/v1/vep105_GRCh38.mt')
+
+    # select matrix down to that one donor
+    donor_mt = mt.filter_cols(mt.s == cpg_id)
+
+    logging.info('Number of total variants: {}'.format(donor_mt.count()[0]))
+
+    # check this file in the future (GENCODE??)
+    # get gene body position (start and end) and build interval
+    # include variants up to 10kb up- and downstream
+    gene_file = (
+        'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/gene_location_files/GRCh38_geneloc_chr'
+        + chrom
+        + '.tsv'
+    )
+    gene_df = pd.read_csv(gene_file, sep='\t', index_col=0)
+    interval_start = float(gene_df[gene_df['gene_name'] == gene_name]['start']) - 10000
+    interval_end = float(gene_df[gene_df['gene_name'] == gene_name]['end']) + 10000
+
+    # get gene-specific genomic interval
+    gene_interval = (
+        'chr' + chrom + ':' + str(interval_start) + '-' + str(interval_end)
+    )  # 'chr22:23219960-23348287'
```

Comment:
```suggestion
    gene_interval = f'chr{chrom}:{interval_start}-{interval_end}'
    # 'chr22:23219960-23348287'
```
Woooo f-strings! They are sexy, _and_ will automatically do type conversions so you don't need to wrap objects in `str()`